views/session-form.hbs

```diff
@@ -0,0 +1,48 @@
+<h1>{{title}}</h1>
+<div class="container">
+    <form action="/scores" method="POST">
+        <div class="form-group">
+            <label for="game">Choose your game:</label>
+                <select multiple name="game" id="game" class="form-control" size="8">
+                    {{#each games}}
+                    <option value={{id}}>{{name}}</option>
+                    {{/each}}
+                    {{!-- <option value="">s</option>
+                    <option value="">sd</option>
```

Comment:
lets take out these test lines before we merge

Reply:
Good call - will do.